src/components/scenes/CryptoExchangeScene.js

```diff
@@ -377,8 +373,8 @@ export const CryptoExchangeScene = connect<StateProps, DispatchProps, {}>(
       dispatch(selectWalletForExchange(walletId, currencyCode, direction))
       dispatch(updateMostRecentWalletsSelected(walletId, currencyCode))
     },
-    exchangeMax() {
-      dispatch(exchangeMax())
+    async exchangeMax() {
+      await dispatch(exchangeMax())
```

Comment:
Was ESLint yelling about this?

Reply:
No, it's a genuine change - before it returned `void`, but now it returns `Promise<void>` so we can show a spinner until it's done working.